packages/domain/lib/src/models/review_message/review_message.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:domain/src/models/review_message/review_user.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'review_message.g.dart';
+
+@JsonSerializable()
+class ReviewMessage {
+  @JsonKey(name: 'comment')
+  final String? message;
```

Comment:
why do you use the word "message" instead of "comment"?)

Reply:
I don't know =) just decided that it will be more comfortable for me)